tasks/main.yml

```diff
@@ -62,3 +62,16 @@
   ansible.builtin.service:
     enabled: yes
     name: nessusagent
+
+- name: Link the Nessus Agent to the CDM Nessus server
+  ansible.builtin.command:
+    argv:
+      - /opt/nessus_agent/sbin/nessuscli
```

Comment:
Should this use the value of `install_directory`?

Reply:
See commit 1bbe524.

Reply:
Thanks for taking care of that!